mep/people/management/commands/export_members.py

```diff
@@ -85,8 +86,10 @@ def get_object_data(self, obj):
         if obj.death_year:
             data["death_year"] = obj.death_year
         # set for unique, list for json serialization
-        data["membership_years"] = list(
-            set(d.year for d in obj.account_set.first().event_dates)
+        data["membership_years"] = (
```

Comment:
instead of making this conditional on whether the person has an account, can we just skip based on whether membership years is in the list of fields to be exported? that isn't relevant for the creator export, is it? (although I know there is some overlap)